src/ds/d400/d400-device.cpp

```diff
@@ -109,6 +109,10 @@ namespace librealsense
 
     void d400_device::enter_update_state() const
     {
+        if( _pid == ds::RS421_PID &&
+            !get_context()->get_settings().nested( "enable-d421-fw-update" ).default_value( false ) )
```

Comment:
How can we activate this using the RS-Viewer?

Reply:
We should add `"enable-d421-fw-update":true` to the context settings in the config file.